cpp/ql/test/library-tests/dataflow/DefaultTaintTracking/annotate_sinks_only/tainted.expected

```diff
@@ -0,0 +1,2 @@
+| defaulttainttracking.cpp:190:14:190:24 | // $ ast,ir | Missing result:ir= |
+| defaulttainttracking.cpp:193:14:193:24 | // $ ast,ir | Missing result:ir= |
```

Comment:
We like keeping the .expected files empty when using inline expectation tests. You can add `MISSING: ir` to the test to make these lines to away.